src/ja/common/proxy/ssh.py

```diff
@@ -16,21 +17,21 @@ class ISSHConnection(ABC):
     """
 
     @abstractmethod
-    def send_server_command(self, command: ServerCommand) -> ServerResponse:
+    def send_server_command(self, command: ServerCommand) -> Response:
```

Comment:
Now that there is only one Response class, please unify send_server_command and send_worker_command.
I only created two separate methods due to the limitations imposed imposed by mypy.